std_msgs/Header header
uint32 my_number
trajectory_generator/Path[] paths